logstash-core/lib/logstash/outputs/base.rb

```diff
@@ -54,36 +61,59 @@ def receive(event)
 
   public
   def worker_setup
+    # TODO: Remove this branch, delete this function
     if @workers == 1
+      @is_multi_worker = false
```

Comment:
I'm not sure if it's worth defining this variable it's not used anywhere.

In https://github.com/elastic/logstash/pull/4254/files#diff-1646a27fa8dcf9f75d0b65aeaa7c9047R100 it is defined again